### ISSUE TITLE
[igCurrencyEditor] When we clear the value, the value is set to minValue,  but the spinDown button is enabled.

### ISSUE THREAD

**Issue by Marina-L-Stoyanova** [IgniteUI/ignite-ui#895]
### Description  
When we clear the value of the editor through the clear button, the value is set to the minValue, but the spinDown button is enabled.  

### Steps to reproduce  

1. Create a sample with the following code: 
```
$("#editor").igCurrencyEditor({       
         displayFactor : 1,
         maxValue: 8,
         minValue: 5,
         spinDelta: 1,
         buttonType: "spin, clear"
    });
```
2. Enter value between the minValue and the maxValue. For example 6.
3. Blur the editor.
4. Click on the clear button.

### Result  
The spinDown button is enabled.

### Expected result  
The spinDown button should be disables as the value equals the minValue..

**Comment by dafo** [comment]
This seems to be fixed after fixing #894

**Comment by dafo** [comment]
This should be tested alongside #894
